AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/holder/ReceiverRegistrationHolder.java

```diff
@@ -34,14 +34,13 @@
 import com.sun.codemodel.JExpression;
 import com.sun.codemodel.JFieldVar;
 
-public class ReceiverRegistrationHolder<T extends EComponentHolder & HasReceiverRegistration> {
+public class ReceiverRegistrationHolder extends GeneratedClassHolderDecorator<EComponentHolder> {
```

Comment:
Why don't you keep `ReceiverRegistrationHolder` generic ? It prevent us from casting and clearly state that the `holder` must implement `HasReceiverRegistration`.